packages/belos/test/GCRODR/test_gcrodr_complex_hb.cpp

```diff
@@ -234,7 +231,7 @@ int main(int argc, char *argv[]) {
     delete [] rowind;
     delete [] cvals;
     // Test for failures
-    if ( ret!=Belos::Converged || norm_failure || numIters1 < numIters2 || numIters1 < numIters3 ) {
+    if ( ret!=Belos::Converged || norm_failure || numIters1 < numIters2 || numIters2 < numIters3 ) {
```

Comment:
This is also a non-backward compatible change?

Reply:
This is a test.

Reply:
@bartlettroscoe @mhoemmen This test started failing after the correction to the GEEV/GGEVX interface.  That is because the assumptions made by the test are false and needed to be corrected.

Reply:
That is good.  At least a test had to change (sorry I missed this was in the /test/ directory).  But there are not test cases for when `*info != 0`?